content/docs/concepts/policies-and-telemetry/istio-metrics.md

```diff
@@ -0,0 +1,99 @@
+
+## Metrics
+
+### Request Count
+
+This is a COUNTER metric incremented for a new request to Istio.
+It is labeled based on source service, source version, destination service, destination version, connection mTLS and response code of the request.
+This is exported by default by prometheus adapter and can be configured to be exported by other mixer adapters.
+
+### Request Duration
+
+This is a DISTRIBUTION metric which measures the duration of the request.
+This metric is obtained from envoy proxy. It is labeled based on source service, source version, destination service, destination version, connection mTLS and response code of the request. This is exported by default by prometheus adapter using explicit bucket configuration of [0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10].
+It can be configured to be exported by other mixer adapters.
+
+### Request Size
+
+This is a DISTRIBUTION metric which measures the size of the http request’s body size.
+This metric is obtained from envoy proxy. It is labeled based on source service, source version, destination service, destination version, connection mTLS and response code of the request. This is exported by default by prometheus adapter using exponential buckets with configuration of number of finite buckets is equal to 8, scale of 1 and growth factor of 10.
+It can be configured to be exported by other mixer adapters.
+
+### Response Size
+
+This is a DISTRIBUTION metric which measures the size of the http response body size.
+This metric is obtained from envoy proxy. It is labeled based on source service, source version, destination service, destination version, connection mTLS and response code of the request. This is exported by default by prometheus adapter using exponential buckets with configuration of number of finite buckets is equal to 8, scale of 1 and growth factor of 10.
+It can be configured to be exported by other mixer adapters.
+
+### Tcp Byte Sent
+
+This is a COUNTER metric which measures the size of total bytes sent during response in case of a TCP connection.
+This metric is obtained from envoy proxy. It is labeled based on source service, source version, destination service, destination version and connection mTLS of the request.
+This is exported by default by prometheus adapter and can be exported by other mixer adapters.
+
+### Tcp Byte Received
+
+This is a COUNTER metric which measures the size of total bytes received during request in case of a TCP connection.
+This metric is obtained from envoy proxy. It is labeled based on source service, source version, destination service, destination version and connection mTLS of the request.
+This is exported by default by prometheus adapter and can be exported by other mixer adapters.
+
+## Labels
+
+### Source Service
+
+This identifies the source service responsible for an incoming request.
+This label is obtained from kubernetes cluster metadata. This is also the FQDN for a source service.
```

Comment:
Let's remove the "obtained from" sentence and add a "Default value expression" `####` or whatever that shows the attribute expression.

Something like:

#### Default Attribute Expression for Label
`source_service: source.service | "unknown"`

same for all following labels.

Reply:
Fixed